targets/python/templates/PlayFabJson.py.ejs

```diff
@@ -0,0 +1,25 @@
+def serialize_instance(obj):
+    d = { '__classname__' : type(obj).__name__ }
+    d.update(dir(obj)) 
+    # o = vars(obj.__dict__)
+    # d.update(o) 
+    # d = { '__classname__' : type(obj).__name__, 'vars' : dir(obj) }
+    # d = { '__classname__' : dir(obj) }
+    return d
+
+classes = {
+<% for(var i in datatype.properties) { var property = datatype.properties[i];
+%>
+<% } %>
+}
+
+def unserialize_object(d):
+    clsname = d.pop('__classname__', None)
+    if clsname:
+        cls = classes[clsname]
+        obj = cls.__new__(cls)  # make instance without calling __init__
+        for key, value in d.items():
+            setattr(obj, key, value)
+            return obj
```

Comment:
Same as above, shouldn't this be outside the loop?

Reply:
Probably.